.travis.yml

```diff
@@ -28,17 +29,12 @@ matrix:
       - ./scripts/start_docker_bblfsh.sh
       - ./scripts/install_python.sh
       - ./scripts/get_apache_spark.sh "2.2.0" "2.7" || travis_terminate 1
-    install:
-      - make build
+      - tar -xzf src/test/resources/weighted-minhash/csv.tar.gz -C src/test/resources/weighted-minhash/
     script:
       # start dependencies
       - VIRTUAL_ENV=None IN_BACKGROUND=1 ./feature_extractor
-      # just check that commands don't fail
-      - ./hash src/test/resources/siva || travis_terminate 1
-      - ./query ./src/test/resources/LICENSE
-      - ./report
-      - go get ./src/main/go/... || true
-      - go run ./src/main/go/query.go ./src/test/resources/LICENSE
```

Comment:
Sorry if I missed something, but what is the reason to remove tests for go-based duplication detection?

It might be ok to move them to different profile to optimize CI speed, or move to the next one, with external Apache Spark, but I would prefer keeping those somewhere.

Reply:
we still run go-cli and all others here:
https://github.com/smacker/gemini/blob/3ac7c7630a3c5a8912e5d0735cc3b30a7b8ee66f/.travis.yml#L89

I don't see any benefits to run it on local spark first and then on spark in cluster mode. Imo, cluster mode should be enough.

Reply:
On Go 👍 , on Cluster mode - avoid having Spark Standalone installed/unpacked would just cut some CI some.